app/controllers/organizations/staff/staff_controller.rb

```diff
@@ -1,43 +1,32 @@
 class Organizations::Staff::StaffController < Organizations::BaseController
-  before_action :set_staff_account, only: [:deactivate, :activate, :update_activation]
+  before_action :set_staff, only: [:update_activation]
 
   layout "dashboard"
 
   def index
-    authorize! StaffAccount, context: {organization: Current.organization}
+    authorize! User, context: {organization: Current.organization}
 
-    @staff_accounts = authorized_scope(StaffAccount.all)
+    @staff = authorized_scope(User.staff)
   end
 
-  def deactivate
-    @staff_account.deactivate
-    respond_to do |format|
-      format.html { redirect_to staff_staff_index_path, notice: t(".success") }
-      format.turbo_stream { render "organizations/staff/staff/update" }
+  def update_activation
+    if @staff.deactivated_at
+      @staff.activate
+    else
+      @staff.deactivate
     end
-  end
 
-  def activate
-    @staff_account.activate
     respond_to do |format|
       format.html { redirect_to staff_staff_index_path, notice: t(".success") }
-      format.turbo_stream { render "organizations/staff/staff/update" }
-    end
-  end
-
-  def update_activation
-    if @staff_account.deactivated_at
-      activate
-    else
-      deactivate
+      format.turbo_stream
```

Comment:
Do we want to flash a notice with this turbostream as well? Like we do in the `UserRolesController`? Although not sure how best to separate the messages for activation vs deactivation.

Reply:
Also I noticed that there is a lingering reference to `staff_accounts` in the `config/locales/en.yml` file?

Reply:
Not sure if the flash messages were working before your changes, but when I check out your branch they don't seem to be working.

Reply:
Yeah good catch - adding the flash to `app/views/organizations/staff/staff/update_activation.turbo_stream.erb` should fix that

Reply:
Created a ticket on the kanban for adding the flash, since it's not in the scope of this PR.

Reply:
Added flash a76573f9b53f

Removed staff_account refs in yml b68048665bf3

@kasugaijin I added the flash in the controller. Do we actually define flash messages in the view file anywhere in the app currently? I'm not sure of the benefit of doing so, but I think that should be a separate PR to refactor for the whole app if there is a reason.

Reply:
Yes it's better to stay consistent.